src/server/transaction.h

```diff
@@ -392,12 +395,11 @@ class Transaction {
 
   enum CoordinatorState : uint8_t {
     COORD_SCHED = 1,
-    COORD_EXEC = 2,
 
-    COORD_EXEC_CONCLUDING = 1 << 2,  // Whether its the last hop of a transaction
-    COORD_BLOCKED = 1 << 3,
-    COORD_CANCELLED = 1 << 4,
-    COORD_OOO = 1 << 5,
+    COORD_CONCLUDING = 1 << 1,  // Whether its the last hop of a transaction
```

Comment:
No, it's not. With multi, we set `multi_->concluding` to log to the journal

> // Whether its the last hop of a transaction

I even think this comment is accurate, because previously it was the last hop of _a command_

Reply:
If this comment is accurate, it should also mark the conclusion of EXEC (i.e. the whole transaction).
in any case, lgtm